src/sections/Facet/Facet.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Field from "../../components/Field/FieldContainer";
+import Checkbox from "styled-components"
+import specificationFacetConfig from "../../configs/specificationFacetsConfig";
+import Button from "@material-ui/core/Button";
+import { Label } from "@material-ui/icons";
+import ListModal from "../../components/ListModal";
+import toolbarConfig from "../../configs/appToolbar";
+import "./Facet.css"
```

Comment:
It seems that line 5-9 are unnecessary imports

Reply:
Yeah, you're right. Just the css file changes the color but we don't have to keep it that way. I just thought it might make sense to have the styling file and the import in case we want to change something there.